polaris.shopify.com/src/styles/globals.scss

```diff
@@ -342,3 +357,34 @@ button {
 .tip-banner__header h4 {
   margin: 0 !important;
 }
+
+.card {
+  background-color: var(--surface);
+  box-shadow: var(--card-shadow);
+  border-radius: var(--border-radius-400-new);
+  padding: var(--card-padding-md);
+}
+
+.card-loose {
+  background-color: var(--surface);
+  box-shadow: var(--card-shadow);
+  border-radius: var(--border-radius-400-new);
+  padding: var(--card-padding-lg);
+}
+
+.text-extra-padding {
+  padding: var(--card-padding-md);
+}
+
+.directive {
```

Comment:
Fast follow: let's break this out into it's own component and update any places we're using `<Box className="directive">` for the new component before we add anymore documentation